src/jquery.js

```diff
@@ -40,6 +40,89 @@ function init() {
   store.commit('app/setKeypressListener', () => keypressListener);
 }
 
+// returns true if bit `value` of `test` is true
+function hasBitsSet(test, value) {
+  return (test & (1 << value)) === Math.pow(2, value);
```

Comment:
why not use 1 << on right side too?

Reply:
I've just always used `Math.pow()` this way, if that's what you're asking.

Reply:
it's just weird. 1 << is orders of magnitude faster than Math.pow because it's an intrinsic CPU instruction, whereas Math.pow is a generalized power function. I would prefer you to use one or the other, but not both.

Reply:
I've also been informed that there is now a ** operation so you know longer need to call Math

https://mariusschulz.com/blog/the-exponentiation-operator-in-javascript